### ISSUE TITLE
Crash on 'New Material'

### ISSUE THREAD

**Issue by sourozowy** [BlenderNPR/BEER#4]
I crash when I create a new material with the Malt render engine enabled. using version "Don't create pipeline on module initialization"
```

Traceback (most recent call last):
  File "D:\Steam Games\steamapps\common\Blender\2.83\python\lib\site-packages\OpenGL\latebind.py", line 43, in __call__
    return self._finalCall( *args, **named )
TypeError: 'NoneType' object is not callable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\MaltPipeline.py", line 104, in depsgraph_update
    setup_all_ids()
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\MaltPipeline.py", line 91, in setup_all_ids
    setup_parameters(bpy.data.scenes)
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\MaltPipeline.py", line 71, in setup_parameters
    pipeline_parameters = get_pipeline().get_parameters()
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\MaltPipeline.py", line 22, in get_pipeline
    __PIPELINE__ = PipelineTest()
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\Malt\PipelineTest.py", line 89, in __init__
    super().__init__()
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\Malt\Pipeline.py", line 71, in __init__
    self.quad = Mesh(positions, indices)
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\Malt\Mesh.py", line 23, in __init__
    glGenBuffers(1, self.EBO)
  File "D:\Steam Games\steamapps\common\Blender\2.83\python\lib\site-packages\OpenGL\latebind.py", line 47, in __call__
    return self._finalCall( *args, **named )
  File "D:\Steam Games\steamapps\common\Blender\2.83\python\lib\site-packages\OpenGL\wrapper.py", line 678, in wrapperCall
    raise err
  File "D:\Steam Games\steamapps\common\Blender\2.83\python\lib\site-packages\OpenGL\wrapper.py", line 671, in wrapperCall
    result = wrappedOperation( *cArguments )
  File "D:\Steam Games\steamapps\common\Blender\2.83\python\lib\site-packages\OpenGL\platform\baseplatform.py", line 415, in __call__
    return self( *args, **named )
  File "D:\Steam Games\steamapps\common\Blender\2.83\python\lib\site-packages\OpenGL\error.py", line 234, in glCheckError
    baseOperation = baseOperation,
OpenGL.error.GLError: GLError(
        err = 1280,
        description = b'invalid enumerant',
        baseOperation = glGenBuffers,
        pyArgs = (
                1,
                <BlenderMalt.Malt.GL.c_long_Array_1 object at 0x0000022AF98569C8>,
        ),
        cArgs = (
                1,
                <BlenderMalt.Malt.GL.c_long_Array_1 object at 0x0000022AF98569C8>,
        ),
        cArguments = (
                1,
                <BlenderMalt.Malt.GL.c_long_Array_1 object at 0x0000022AF98569C8>,
        )
)
Traceback (most recent call last):
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\MaltRenderEngine.py", line 34, in __init__
    self.pipeline = MaltPipeline.get_pipeline().__class__()
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\Malt\PipelineTest.py", line 89, in __init__
    super().__init__()
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\Malt\Pipeline.py", line 71, in __init__
    self.quad = Mesh(positions, indices)
  File "C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\BlenderMalt\Malt\Mesh.py", line 23, in __init__
    glGenBuffers(1, self.EBO)
  File "D:\Steam Games\steamapps\common\Blender\2.83\python\lib\site-packages\OpenGL\latebind.py", line 43, in __call__
    return self._finalCall( *args, **named )
  File "D:\Steam Games\steamapps\common\Blender\2.83\python\lib\site-packages\OpenGL\wrapper.py", line 678, in wrapperCall
    raise err
  File "D:\Steam Games\steamapps\common\Blender\2.83\python\lib\site-packages\OpenGL\wrapper.py", line 671, in wrapperCall
    result = wrappedOperation( *cArguments )
  File "D:\Steam Games\steamapps\common\Blender\2.83\python\lib\site-packages\OpenGL\error.py", line 234, in glCheckError
    baseOperation = baseOperation,
OpenGL.error.GLError: GLError(
        err = 1280,
        description = b'invalid enumerant',
        baseOperation = glGenBuffers,
        pyArgs = (
                1,
                <BlenderMalt.Malt.GL.c_long_Array_1 object at 0x0000022AF97A3F48>,
        ),
        cArgs = (
                1,
                <BlenderMalt.Malt.GL.c_long_Array_1 object at 0x0000022AF97A3F48>,
        ),
        cArguments = (
                1,
                <BlenderMalt.Malt.GL.c_long_Array_1 object at 0x0000022AF97A3F48>,
        )
)

location: <unknown location>:-1
```

This line is repeated quite a number of times:

```

WARN (bpy.rna): C:\b\buildbot-worker-windows\windows_lts_283\blender.git\source\blender\python\intern\bpy_rna.c:1479 pyrna_enum_to_py: current value '0' matches no enum in 'SpaceNodeEditor', '(null)', 'tree_type`

```

And then, actually clicking New Material closes blender, Powershell gives me this:

```

Error   : EXCEPTION_ACCESS_VIOLATION
Address : 0x00007FF61B79B184
Module  : D:\Steam Games\steamapps\common\Blender\blender.exe
PS C:\Windows\System32\WindowsPowerShell\v1.0> ddddddfasfasfaf
ddddddfasfasfaf : The term 'ddddddfasfasfaf' is not recognized as
the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify
that the path is correct and try again.
At line:1 char:1
+ ddddddfasfasfaf
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (ddddddfasfasfaf:St
   ring) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException`
```

 "ddddddfasfasfaf" looks like something i'd type but it was a new blender file with default cube.

**Comment by pragma37** [comment]
It's the same issue from your other thread. #3 

What happens if you turn on the viewport rendering on an empty scene, does it work? If you change the background color in the World settings does the background color changes?
Have you restarted Blender after installation?

Please, use a non-Steam version for testing https://www.blender.org/download/Blender2.83/blender-2.83.3-windows64.zip/ and post your System Info log from Blender.

**Comment by sourozowy** [comment]
Have I not been pasting a system info log? I can't find any google results that help. One result even says Blender has disabled saving the error log. I was going to embed a video here of me running through the steps that you wanted but git isn't letting me do it. https://cdn.discordapp.com/attachments/729127664434282557/738844009937764362/2020-07-31_20-35-18-1.webm if you're in the discord

**Comment by pragma37** [comment]
Inside Blender, Help > Save System Info

**Comment by sourozowy** [comment]
Someone in the discord has confirmed they're crashing in blender 2.9

system info

```
==========================================
= Blender 2.90.0 Beta System Information =
==========================================


Blender:
==========================================

version: 2.90.0 Beta, branch: master, commit date: 2020-07-29 22:17, hash: 699cefefd009, type: Release
build date: 2020-07-29, 21:20:07
platform: Windows
binary path: 'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\blender.exe'
build cflags: /W3 /w34062 /w34115 /w34189 /wd4018 /wd4146 /wd4065 /wd4127 /wd4181 /wd4200 /wd4244 /wd4267 /wd4305 /wd4800 /wd4828 /wd4996 /wd4661 /we4013 /we4133 /we4431 /w35038 /DWIN32 /D_WINDOWS /W3 /nologo /J /Gd /MP /bigobj -openmp     
build cxxflags: /W3 /w34062 /w34115 /w34189 /wd4018 /wd4146 /wd4065 /wd4127 /wd4181 /wd4200 /wd4244 /wd4267 /wd4305 /wd4800 /wd4828 /wd4996 /wd4661 /we4013 /we4133 /we4431 /w35038 /DWIN32 /D_WINDOWS /W3 /GR /EHsc /nologo /J /Gd /MP /EHsc /bigobj /permissive- /Zc:twoPhase- -openmp /std:c++17     
build linkflags: /MACHINE:X64  /SUBSYSTEM:CONSOLE /STACK:2097152 /ignore:4049 /ignore:4217 /ignore:4221
build system: CMake

Python:
==========================================

version: 3.7.7 (default, Jun 13 2020, 11:11:23) [MSC v.1916 64 bit (AMD64)]
file system encoding: utf-8:surrogatepass
paths:
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\scripts\\startup'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\scripts\\modules'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\python37.zip'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\python\\DLLs'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\python\\lib'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\python'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\python\\lib\\site-packages'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\scripts\\freestyle\\modules'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\scripts\\addons\\modules'
	'C:\\Users\\Joseph\\AppData\\Roaming\\Blender Foundation\\Blender\\2.90\\scripts\\addons\\modules'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\scripts\\addons'
	'C:\\Users\\Joseph\\AppData\\Roaming\\Blender Foundation\\Blender\\2.90\\scripts\\addons'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\scripts\\addons_contrib'

Python (External Binary):
==========================================

binary path: 'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\python\\bin\\python.EXE'
version: Python 3.7.7

Directories:
==========================================

scripts:
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\scripts\\modules'
	'C:\\Users\\Joseph\\Desktop\\blender-2.90.0-699cefefd009-windows64\\2.90\\scripts'
	'C:\\Users\\Joseph\\AppData\\Roaming\\Blender Foundation\\Blender\\2.90\\scripts'
user scripts: 'C:\\Users\\Joseph\\AppData\\Roaming\\Blender Foundation\\Blender\\2.90\\scripts'
pref scripts: None
datafiles: 'C:\\Users\\Joseph\\AppData\\Roaming\\Blender Foundation\\Blender\\2.90\\datafiles\\'
config: 'C:\\Users\\Joseph\\AppData\\Roaming\\Blender Foundation\\Blender\\2.90\\config\\'
scripts : 'C:\\Users\\Joseph\\AppData\\Roaming\\Blender Foundation\\Blender\\2.90\\scripts\\'
autosave: 'C:\\Users\\Joseph\\AppData\\Roaming\\Blender Foundation\\Blender\\2.90\\autosave\\'
tempdir: 'C:\\Users\\Joseph\\AppData\\Local\\Temp\\blender_a07836\\'

FFmpeg:
==========================================

avcodec:   '58, 54, 100'
avdevice:  '58,  8, 100'
avformat:  '58, 29, 100'
avutil:    '56, 31, 100'
swscale:   ' 5,  5, 100'

SDL:
==========================================

Version: 2.0.12
Loading method: linked (WITH_SDL_DYNLOAD=OFF)

Other Libraries:
==========================================

OpenColorIO:  1,  1,  1
OpenImageIO:  2,  1, 15
OpenShadingLanguage:  1, 10, 10
OpenSubdiv:  0,  0,  0
OpenVDB:  7,  0,  0
Alembic:  1,  7, 12
USD:  0, 20,  5

OpenGL:
==========================================

renderer:	'Intel(R) Iris(R) Pro Graphics 580'
vendor:		'Intel'
version:	'4.5.0 - Build 26.20.100.8141'
extensions:
	GL_3DFX_texture_compression_FXT1
	GL_AMD_depth_clamp_separate
	GL_AMD_vertex_shader_layer
	GL_AMD_vertex_shader_viewport_index
	GL_ARB_ES2_compatibility
	GL_ARB_ES3_1_compatibility
	GL_ARB_ES3_compatibility
	GL_ARB_arrays_of_arrays
	GL_ARB_base_instance
	GL_ARB_blend_func_extended
	GL_ARB_buffer_storage
	GL_ARB_cl_event
	GL_ARB_clear_buffer_object
	GL_ARB_clear_texture
	GL_ARB_clip_control
	GL_ARB_color_buffer_float
	GL_ARB_compressed_texture_pixel_storage
	GL_ARB_compute_shader
	GL_ARB_conditional_render_inverted
	GL_ARB_conservative_depth
	GL_ARB_copy_buffer
	GL_ARB_copy_image
	GL_ARB_cull_distance
	GL_ARB_debug_output
	GL_ARB_depth_buffer_float
	GL_ARB_depth_clamp
	GL_ARB_depth_texture
	GL_ARB_derivative_control
	GL_ARB_direct_state_access
	GL_ARB_draw_buffers
	GL_ARB_draw_buffers_blend
	GL_ARB_draw_elements_base_vertex
	GL_ARB_draw_indirect
	GL_ARB_draw_instanced
	GL_ARB_enhanced_layouts
	GL_ARB_explicit_attrib_location
	GL_ARB_explicit_uniform_location
	GL_ARB_fragment_coord_conventions
	GL_ARB_fragment_layer_viewport
	GL_ARB_fragment_program
	GL_ARB_fragment_program_shadow
	GL_ARB_fragment_shader
	GL_ARB_fragment_shader_interlock
	GL_ARB_framebuffer_no_attachments
	GL_ARB_framebuffer_object
	GL_ARB_framebuffer_sRGB
	GL_ARB_geometry_shader4
	GL_ARB_get_program_binary
	GL_ARB_get_texture_sub_image
	GL_ARB_gl_spirv
	GL_ARB_gpu_shader5
	GL_ARB_gpu_shader_fp64
	GL_ARB_half_float_pixel
	GL_ARB_half_float_vertex
	GL_ARB_indirect_parameters
	GL_ARB_instanced_arrays
	GL_ARB_internalformat_query
	GL_ARB_internalformat_query2
	GL_ARB_invalidate_subdata
	GL_ARB_map_buffer_alignment
	GL_ARB_map_buffer_range
	GL_ARB_multi_bind
	GL_ARB_multi_draw_indirect
	GL_ARB_multisample
	GL_ARB_multitexture
	GL_ARB_occlusion_query
	GL_ARB_occlusion_query2
	GL_ARB_pipeline_statistics_query
	GL_ARB_pixel_buffer_object
	GL_ARB_point_parameters
	GL_ARB_point_sprite
	GL_ARB_polygon_offset_clamp
	GL_ARB_post_depth_coverage
	GL_ARB_program_interface_query
	GL_ARB_provoking_vertex
	GL_ARB_query_buffer_object
	GL_ARB_robust_buffer_access_behavior
	GL_ARB_robustness
	GL_ARB_robustness_isolation
	GL_ARB_sample_shading
	GL_ARB_sampler_objects
	GL_ARB_seamless_cube_map
	GL_ARB_seamless_cubemap_per_texture
	GL_ARB_separate_shader_objects
	GL_ARB_shader_atomic_counter_ops
	GL_ARB_shader_atomic_counters
	GL_ARB_shader_bit_encoding
	GL_ARB_shader_draw_parameters
	GL_ARB_shader_group_vote
	GL_ARB_shader_image_load_store
	GL_ARB_shader_image_size
	GL_ARB_shader_objects
	GL_ARB_shader_precision
	GL_ARB_shader_stencil_export
	GL_ARB_shader_storage_buffer_object
	GL_ARB_shader_subroutine
	GL_ARB_shader_texture_image_samples
	GL_ARB_shading_language_100
	GL_ARB_shading_language_420pack
	GL_ARB_shading_language_packing
	GL_ARB_shadow
	GL_ARB_spirv_extensions
	GL_ARB_stencil_texturing
	GL_ARB_sync
	GL_ARB_tessellation_shader
	GL_ARB_texture_barrier
	GL_ARB_texture_border_clamp
	GL_ARB_texture_buffer_object
	GL_ARB_texture_buffer_object_rgb32
	GL_ARB_texture_buffer_range
	GL_ARB_texture_compression
	GL_ARB_texture_compression_bptc
	GL_ARB_texture_compression_rgtc
	GL_ARB_texture_cube_map
	GL_ARB_texture_cube_map_array
	GL_ARB_texture_env_add
	GL_ARB_texture_env_combine
	GL_ARB_texture_env_crossbar
	GL_ARB_texture_env_dot3
	GL_ARB_texture_filter_anisotropic
	GL_ARB_texture_float
	GL_ARB_texture_gather
	GL_ARB_texture_mirror_clamp_to_edge
	GL_ARB_texture_mirrored_repeat
	GL_ARB_texture_multisample
	GL_ARB_texture_non_power_of_two
	GL_ARB_texture_query_levels
	GL_ARB_texture_query_lod
	GL_ARB_texture_rectangle
	GL_ARB_texture_rg
	GL_ARB_texture_rgb10_a2ui
	GL_ARB_texture_stencil8
	GL_ARB_texture_storage
	GL_ARB_texture_storage_multisample
	GL_ARB_texture_swizzle
	GL_ARB_texture_view
	GL_ARB_timer_query
	GL_ARB_transform_feedback2
	GL_ARB_transform_feedback3
	GL_ARB_transform_feedback_instanced
	GL_ARB_transform_feedback_overflow_query
	GL_ARB_transpose_matrix
	GL_ARB_uniform_buffer_object
	GL_ARB_vertex_array_bgra
	GL_ARB_vertex_array_object
	GL_ARB_vertex_attrib_64bit
	GL_ARB_vertex_attrib_binding
	GL_ARB_vertex_buffer_object
	GL_ARB_vertex_program
	GL_ARB_vertex_shader
	GL_ARB_vertex_type_10f_11f_11f_rev
	GL_ARB_vertex_type_2_10_10_10_rev
	GL_ARB_viewport_array
	GL_ARB_window_pos
	GL_ATI_separate_stencil
	GL_EXT_abgr
	GL_EXT_bgra
	GL_EXT_blend_color
	GL_EXT_blend_equation_separate
	GL_EXT_blend_func_separate
	GL_EXT_blend_minmax
	GL_EXT_blend_subtract
	GL_EXT_clip_volume_hint
	GL_EXT_compiled_vertex_array
	GL_EXT_direct_state_access
	GL_EXT_draw_buffers2
	GL_EXT_draw_range_elements
	GL_EXT_fog_coord
	GL_EXT_framebuffer_blit
	GL_EXT_framebuffer_multisample
	GL_EXT_framebuffer_object
	GL_EXT_geometry_shader4
	GL_EXT_gpu_program_parameters
	GL_EXT_gpu_shader4
	GL_EXT_multi_draw_arrays
	GL_EXT_packed_depth_stencil
	GL_EXT_packed_float
	GL_EXT_packed_pixels
	GL_EXT_polygon_offset_clamp
	GL_EXT_rescale_normal
	GL_EXT_secondary_color
	GL_EXT_separate_specular_color
	GL_EXT_shader_framebuffer_fetch
	GL_EXT_shader_integer_mix
	GL_EXT_shadow_funcs
	GL_EXT_stencil_two_side
	GL_EXT_stencil_wrap
	GL_EXT_texture3D
	GL_EXT_texture_array
	GL_EXT_texture_compression_s3tc
	GL_EXT_texture_edge_clamp
	GL_EXT_texture_env_add
	GL_EXT_texture_env_combine
	GL_EXT_texture_filter_anisotropic
	GL_EXT_texture_integer
	GL_EXT_texture_lod_bias
	GL_EXT_texture_rectangle
	GL_EXT_texture_sRGB
	GL_EXT_texture_sRGB_decode
	GL_EXT_texture_shared_exponent
	GL_EXT_texture_snorm
	GL_EXT_texture_storage
	GL_EXT_texture_swizzle
	GL_EXT_timer_query
	GL_EXT_transform_feedback
	GL_IBM_texture_mirrored_repeat
	GL_INTEL_conservative_rasterization
	GL_INTEL_fragment_shader_ordering
	GL_INTEL_framebuffer_CMAA
	GL_INTEL_map_texture
	GL_INTEL_multi_rate_fragment_shader
	GL_INTEL_performance_query
	GL_KHR_blend_equation_advanced
	GL_KHR_blend_equation_advanced_coherent
	GL_KHR_context_flush_control
	GL_KHR_debug
	GL_KHR_no_error
	GL_KHR_texture_compression_astc_ldr
	GL_NV_blend_square
	GL_NV_conditional_render
	GL_NV_primitive_restart
	GL_NV_texgen_reflection
	GL_SGIS_generate_mipmap
	GL_SGIS_texture_edge_clamp
	GL_SGIS_texture_lod
	GL_SUN_multi_draw_arrays
	GL_WIN_swap_hint
	WGL_EXT_swap_control

Implementation Dependent OpenGL Limits:
==========================================

Maximum DrawElements Vertices:	1048576
Maximum DrawElements Indices:	1048576

GLSL:
Maximum Varying Floats:	64
Maximum Vertex Attributes:	16
Maximum Vertex Uniform Components:	4096
Maximum Fragment Uniform Components:	4096
Maximum Vertex Image Units:	32
Maximum Fragment Image Units:	32
Maximum Pipeline Image Units:	192

Cycles:
==========================================


CPU device capabilities: SSE2 SSE3 SSE41 AVX AVX2

OpenCL device capabilities:
Number of platforms: 1
Platform #0
	Platform Name: Intel(R) OpenCL HD Graphics
	Platform Vendor: Intel(R) Corporation
	Platform Version: OpenCL 2.1 
	Platform Profile: FULL_PROFILE
	Platform Extensions: 
	Number of devices: 1
		Device: #0
			Device Name: Intel(R) Iris(R) Pro Graphics 580
			Device Vendor: Intel(R) Corporation
			Device OpenCL C Version: OpenCL C 2.0 
			Device Profile: FULL_PROFILE
			Device Version: OpenCL 2.1 NEO 
			Device Extensions: 
			Device Max clock frequency (MHz): 950
			Device Max compute units: 72
			Device Max work group size: 256

Enabled add-ons:
==========================================

io_scene_fbx (version: (4, 21, 3), path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\io_scene_fbx\__init__.py)
io_anim_bvh (version: (1, 0, 0), path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\io_anim_bvh\__init__.py)
io_mesh_ply (version: (2, 0, 0), path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\io_mesh_ply\__init__.py)
io_scene_obj (version: (3, 8, 0), path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\io_scene_obj\__init__.py)
io_mesh_stl (version: (1, 1, 3), path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\io_mesh_stl\__init__.py)
io_mesh_uv_layout (version: (1, 1, 1), path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\io_mesh_uv_layout\__init__.py)
io_curve_svg (version: UNKNOWN, path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\io_curve_svg\__init__.py)
cycles (version: UNKNOWN, path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\cycles\__init__.py)
io_scene_gltf2 (version: (1, 3, 43), path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\io_scene_gltf2\__init__.py)
node_wrangler (version: (3, 37), path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\node_wrangler.py)
KomikazeAddon (version: ('1', '2'), path: C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.90\scripts\addons\KomikazeAddon\__init__.py)
io_scene_valvesource (version: (3, 0, 3), path: C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.90\scripts\addons\io_scene_valvesource\__init__.py)
OutlineHelper (version: (1, 1, 0), path: C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.90\scripts\addons\OutlineHelper\__init__.py)
bevel-curve-tools-master (version: (0, 1, 1), path: C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.90\scripts\addons\bevel-curve-tools-master\__init__.py)
flip_fluids_addon (version: (1, 0, 7), path: C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.90\scripts\addons\flip_fluids_addon\__init__.py)
spring_bones (version: (0, 9), path: C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.90\scripts\addons\spring_bones.py)
rigify (version: (0, 6, 1), path: C:\Users\Joseph\Desktop\blender-2.90.0-699cefefd009-windows64\2.90\scripts\addons\rigify\__init__.py)
BlenderMalt (version: UNKNOWN, path: C:\Users\Joseph\AppData\Roaming\Blender Foundation\Blender\2.90\scripts\addons\BlenderMalt\__init__.py)

```

**Comment by pragma37** [comment]
Solved on Discord.
It only crased when adding shading node trees, which are not used in Malt.

Fixed the crash in 994465fe43d999ed64718932deb7e3f6f2a55e3c
